arches/app/search/components/ids.py

```diff
@@ -0,0 +1,27 @@
+from arches.app.search.elasticsearch_dsl_builder import Bool, Ids
+from arches.app.search.components.base import BaseSearchFilter
+import json
+
+details = {
+    "searchcomponentid": "f1856bfb-c3c4-4d67-8f23-0aa3eef3a160",
+    "name": "Ids Filter",
+    "icon": "",
+    "modulename": "ids.py",
+    "classname": "IdsFilter",
+    "type": "ids-filter-type",
+    "componentpath": "",
+    "componentname": "ids",
+    "config": {},
+}
+
+
+class IdsFilter(BaseSearchFilter):
+
+    def append_dsl(self, search_query_object, **kwargs):
+        ids = kwargs.get("querystring", None)
```

Comment:
I think this will always come in as a string.  You might need to deserialize first then check 'isinstance'.

Reply:
just pushed a commit